common.js

```diff
@@ -99,9 +100,21 @@ function generateFinalPath (opts) {
   return path.join(opts.out || process.cwd(), generateFinalBasename(opts))
 }
 
-function subOptionWarning (properties, optionName, parameter, value) {
+function info (message, quiet) {
+  if (!quiet) {
+    console.error(message)
```

Comment:
@malept Is there a reason you chose `console.error` instead of `console.info`/`console.log` ?

`console.error` creates a misleading error message (also displayed in red):
```
[ERROR] Packaging app for platform win32 x64 using electron v1.6.10
```